cli/lib/config-checker.js

```diff
@@ -38,14 +38,37 @@ function checkSite(site, multisite = false) {
         } else if (typeof site.disableServiceWorker !== "boolean") {
             reject(`Invalid DjinnJS configuration. The disableServiceWorker value must be a boolean.`);
         } else {
-            console.warn("disableServiceWorker value has been deprecated use serviceWorker instead");
+            console.warn("\ndisableServiceWorker value has been deprecated use serviceWorker instead\n");
             site.serviceWorker = site.disableServiceWorker ? false : true;
         }
 
         if (site.serviceWorker === undefined) {
-            site.serviceWorker = true;
-        } else if (typeof site.serviceWorker !== "boolean") {
-            reject(`Invalid DjinnJS configuration. The serviceWorker value must be a boolean.`);
+            site.serviceWorker = "offline-first";
```

Comment:
I think we could improve this section with refactoring this multi line if statement to use some default values and switch statements.

Reply:
I think the code block above is the same basic structure as well, and could use the same refactoring.

Reply:
I don't disagree that the config check logic should be refactored as switch statements but we can't solely rely on the `typeof` values because of the `null` checks. In JavaScript a value set to `null` has a type of `object` and some of the values support `object` as a valid type alongside `null`

Reply:
That is something that we could address as a single check once. Ultimately, we are setting the `site.serviceWorker` value from this, which this code is only used in the build process, correct? If that is the case, we don't need to block this release for the change, perhaps we could do as a v0.1.1 release unless npm does not allow for sub point releases.